GLAA.Services/LicenceApplication/LicenceApplicationPostDataHandler.cs

```diff
@@ -375,6 +375,17 @@ private void UpdateStatus(Licence licence, LicenceApplicationViewModel model)
                 change.DateCreated = dateTimeProvider.Now();
                 change.Licence = licence;
                 repository.Upsert(change);
+
+                //Update the current Licence for the status
```

Comment:
Kill these kind of comments pls, the code is enough!

Reply:
Removed them both, I guess its quite obvious what we are doing! x)